main/lsp/requests/prepare_rename.cc

```diff
@@ -70,23 +69,30 @@ unique_ptr<ResponseMessage> PrepareRenameTask::runRequest(LSPTypecheckerDelegate
         // Note: Need to correctly type variant here so it goes into right 'slot' of result variant.
         response->result = variant<JSONNullObject, unique_ptr<PrepareRenameResult>>(JSONNullObject());
         auto &queryResponses = result.responses;
-        if (!queryResponses.empty()) {
-            auto resp = move(queryResponses[0]);
+
+        for (auto const &resp : queryResponses) {
```

Comment:
Why are you iterating through the responses? The first response should always be the one you want. Is there a case that this choice is broken for?